pfcpiface/conn.go

```diff
@@ -63,6 +63,102 @@ type PFCPConn struct {
 	shutdown chan struct{}
 
 	metrics.InstrumentPFCP
+	hbMonitor *hbMonitor
+}
+
+type HbTimerStatus uint8
+
+const (
+	CancelHbTimer HbTimerStatus = iota
+	StartHbTimer
+	StopHbTimer
+)
+
+type hbMonitor struct {
+	hbMonCh chan HbTimerStatus
```

Comment:
Can we generalize the handling of all responses to our request instead? Something like this
https://github.com/TheWayYouMakeMeFeel/pfcp-kitchen-sink/blob/master/pkg/pfcp/pfcp.go#L907-L915
https://github.com/TheWayYouMakeMeFeel/pfcp-kitchen-sink/blob/master/cmd/pfcpclient/main.go#L72-L75

Reply:
cc @badhrinathpa

Reply:
Changes in progress.

Reply:
Hi @krsna1729 ,
PFB the link for the changes we made for handling outgoing requests & let us know if we can use the heartbeat parameters in [upf.go (ln:77-79)](https://github.com/clstefi/upf-epc/blob/hbTimer_impl_dev/pfcpiface/upf.go#:~:text=hbMaxRetries%20%20%20uint8,hbRespDuration%20time.Duration) for all the responses.

https://github.com/omec-project/upf-epc/compare/master...clstefi:hbtimer_pfcp_refactoring?expand=1

Reply:
yes without the hb prefix

Reply:
Made the changes and committed it to the PR.